components/brave_vpn/brave_vpn_service.cc

```diff
@@ -262,6 +271,8 @@ void BraveVpnService::OnConnected() {
   }
 
   UpdateAndNotifyConnectionStateChange(ConnectionState::CONNECTED);
+
+  RecordP3A(true);
```

Comment:
Just to confirm, this `OnConnected()` is only used by desktop.
Corresponding call on Android is https://github.com/brave/brave-core/pull/14168/files#diff-dc47b40d1036005af516eb3a98a8c90d7a2653a0eaba755e82238616009853a3R66?